README.md

```diff
@@ -213,6 +213,15 @@ To generate a service key, use the `keys generate` command in the Flow CLI.
 flow keys generate
 ```
 
+## Emulating mainnet and testnet transactions
+The emulator allows you to simulate the execution of transactions as if they were 
+performed on the mainnet or testnet. In order to activate this feature, 
+you must specify the network name for the chain ID flag in the following manner:
+```
+flow emulator --chain-id mainnet
+```
+Please note, the actual execution on the real network may differ.
```

Comment:
```suggestion
Please note, the actual execution on the real network may still differ, due to:
- different execution code
- different handling of signatures
- different block formation
```

Might be worth explaining a bit more about why so that the user has a better idea of what they can expect.